am-lib/src/test/java/integration/uk/gov/hmcts/reform/amlib/FilterResourceIntegrationTest.java

```diff
@@ -329,7 +335,16 @@ void whenExplicitAccessWithDifferentRelationshipParentChildAttributeDiffPermissi
                     JsonPointer.valueOf(CHILD_ATTRIBUTE), ImmutableSet.of(CREATE, READ)))
                 .accessType(EXPLICIT)
                 .build())
-            .relationships(ImmutableSet.of(OTHER_ROLE_NAME, ROLE_NAME))
+            .relationships(ImmutableSet.of(otherRoleName, roleName))
             .build());
     }
+
+    private DefaultPermissionGrant createDefaultPermissionGrant(ResourceDefinition resourceDefinition,
```

Comment:
I think role should be passed as argument as well, otherwise someone who reads tests have to waste time to figure out what role has been used.

Reply:
Makes sense to me :)